app/documents/order_by_articles.rb

```diff
@@ -18,59 +12,30 @@ def title
 
   def body
     each_order_article do |order_article|
-      down_or_page
-
-      rows = []
       dimrows = []
-      each_group_order_article_for(order_article) do |goa|
+      rows = [[
+        GroupOrder.human_attribute_name(:ordergroup),
+        GroupOrderArticle.human_attribute_name(:ordered),
+        GroupOrderArticle.human_attribute_name(:received),
+        GroupOrderArticle.human_attribute_name(:total_price)
+      ]]
+
+      order_article.group_order_articles.each do |goa|
```

Comment:
Would be good to use a function getting the goa's, e.g. if a plugin wants to add an additional field hat needs to be selected from the database.